ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -1772,6 +1772,14 @@ RHHC:
   - return_air_relative_humidity_sensor
   - return_air_relative_humidity_setpoint
 
+EHC:
```

Comment:
Please make the sensor required and add humidification/dehumidification run commands.

Reply:
@shambergoldstein Done

Reply:
since you have both dehumidification and humidification, please change the name to `EDHC`

Reply:
@shambergoldstein Done